### ISSUE TITLE
typescript peer dependency is required

### ISSUE THREAD

**Issue by balovbohdan** [jest-community/eslint-plugin-jest#479]
We have this warning on the version `23.0.4` after running `yarn`:

```
"eslint-plugin-jest > @typescript-eslint/experimental-utils > @typescript-eslint/typescript-estree > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta"
```
There is `"@typescript-eslint/experimental-utils": "^2.5.0"` dependency. Think it should be placed in the `devDependencies`.

**Comment by SimenB** [comment]
duplicate of #467
